src/pudl/etl/__init__.py

```diff
@@ -32,6 +32,7 @@
 default_assets = (
     *load_assets_from_modules([eia_bulk_elec_assets], group_name="eia_bulk_elec"),
     *load_assets_from_modules([epacems_assets], group_name="epacems"),
+    *load_assets_from_modules([pudl.extract.excel], group_name="excel"),
```

Comment:
Are there any assets being defined in `pudl.extract.excel` that need to be loaded here? It looks like the asset factory that's defined in this module is being called to produce the actual assets in the `pudl.extract.eia{860,861,923}` modules, each of which are independently listed below.

I could also not really understand how the dynamic asset graph thing works.